packages/webapp/pages/onboarding.tsx

```diff
@@ -258,6 +273,30 @@ export function OnboardPage(): ReactElement {
       return setActiveScreen(OnboardingStep.Extension);
     }
 
+    const haveSkippedExtension =
+      !options?.clickExtension && activeScreen === OnboardingStep.Extension;
+    const browserName = getCurrentBrowserName();
+    const browserDontHaveExtension = [
+      BrowserName.Safari,
+      BrowserName.Firefox,
+    ].includes(browserName);
+
+    if (
+      isLaptop &&
+      !isCurrentPWA &&
+      canUserInstallDesktop &&
+      activeScreen !== OnboardingStep.InstallDesktop &&
```

Comment:
It's hard to see but doesn't this now prompt it as the first screen?
Also steps beyond this might re-show it now?

Reply:
I've tested and this works as intended. 
At this point of the function there is no chance to get evaluated before last step ✔️